ShinyApps/portal_dashboard/global.R

```diff
@@ -0,0 +1,21 @@
+library(shiny)
+library(shinydashboard)
+library(DT)
+library(plotly)
+library(streamgraph)
+library(forcats)
+
+library(tidyverse)
+library(feather)
+library(lubridate)
+library(plotly)
+library(DT)
+library(kableExtra)
+# library(streamgraph)
+library(ggthemes)
+library(rlang)
+library(synapser)
+
+source("~/R/synapse_helpers.R")
```

Comment:
Are these paths correct? I'd think they'd need to be something like `../../R/R/synapse_helpers.R`, or else maybe the shiny apps should install the syndccutils package from GitHub.

Reply:
> else maybe the shiny apps should install the syndccutils package from GitHub.

Exactly! I'll update the app to install `syndccutils` and remove the `source(...)` call. It probably makes sense to eventually move Shiny apps to one or more different repos (@kdaily would agree); but can keep here for now.

Reply:
I just checked on these files and some of the functions in them are not exported by syndccutils. So if the shiny apps use any of these functions we should either keep the `source()` calls for now or export the relevant functions.

Reply:
I think the only functions the `portal_dashboard` app uses currently are `get_table_df` and `create_synapse_links` (both exported). The `tool_catalog` app is someting that @xindiguo built independently of `syndccutils`, so should be safe. I just noticed that `portal-dashboard` (hyphen, not underscore) is old and should be removed. I'll do some shiny cleanup in a separate PR.